robocop/checkers/misc.py

```diff
@@ -193,7 +193,6 @@
             *** Variables ***
             ${VAR_NO_VALUE}                   # missing value
             ${VAR_WITH_EMPTY}       ${EMPTY}
-            @{MULTILINE_FIRST_EMPTY}
```

Comment:
I believe it should stay - it shows the multiline list variable, that have empty values

Reply:
Yes, I might have removed it by mistake. Good catch, thanks